include/lo2s/perf/event_description.hpp

```diff
@@ -43,9 +47,10 @@ enum class Availability
 struct EventDescription
 {
     EventDescription(const std::string& name, perf_type_id type, std::uint64_t config,
-                     std::uint64_t config1 = 0, double scale = 1, std::string unit = "#")
+                     std::uint64_t config1 = 0, std::set<Cpu> cpus = std::set<Cpu>(),
+                     double scale = 1, std::string unit = "#")
     : name(name), type(type), config(config), config1(config1), scale(scale), unit(unit),
-      availability(Availability::UNAVAILABLE)
+      availability(Availability::UNAVAILABLE), cpus_(cpus)
```

Comment:
I'm not sure how I would go on to rework event_is openable in `src/perf/event_provider.cpp` in a way so that availibility is passed on EventDescrpiption construction.

Reply:
I think it's fine for this PR, but please open a new issue so we can track this.